src/components/banner/banner-images/samahan-members-banner/SAMAHANMembersBanner.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Image from 'next/image';
+
+import { motion } from 'framer-motion';
+
+import { BannerGradient } from '@/components/banner/banner-gradient/BannerGradient';
+import { BannerImageContainer } from '@/components/banner/banner-image-container/BannerImageContainer';
+
+/**
+ *
+ * Use ui/banner.tsx, not this.
+ *
+ */
+
+export interface Props {
+  subtitle?: string;
+  title?: string;
+}
```

Comment:
Please label this correctly

Reply:
How should I label it properly?

Reply:
BannerProps Perhaps? Change "Props" to something more descriptive. This becomes very important especially when you're trying to export the props (assuming this will be used somewhere else). Naming it "Props" will make other people who will work on your code wonder what these Props are for.

Reply:
Ohh okok that makes sense